packages/api/cms-api/src/page-tree/page-tree-read-api.ts

```diff
@@ -92,9 +92,7 @@ export function createReadApi(
                 nodes = sortPreloadedNodes(nodes, sort);
             }
 
-            if (offset !== undefined && limit !== undefined) {
```

Comment:
In my opinion the function should only be called when both offset and limit are defined